### DISCUSSION TITLE
[Tree] 数据不能有 nodeType 否则浏览器控制台会告警

### DISCUSSION THREAD

**Discussion by Otto-J** [arco-design/arco-design-vue#1804] [Q&A]
- [x] I'm sure this does not appear in [the issue list of the repository](https://github.com/arco-design/arco-design-vue/issues)

## 基本信息 
- **依赖包名 及 版本:** @arco-design/web-vue@2.37.4
- **框架版本:** vue3@latest
- **浏览器版本:** chrome106.0.0.0
- **复现地址:** https://stackblitz.com/edit/angular-dbcumn?file=src/App.vue

## 预期结果
控制台应该没有告警，实际有告警

## 复现步骤
1. 打开链接
2. 观察控制台
3. 修改数据 noeType2 改成 nodeType
4. 观察控制台

补充：

如果后续你遇到类似的问题，组件库没有修复的话，前端可以先绕过这部分限制，比如通过递归方法，先修改key，保存数据时候再改回来。

前端总不能卡在这不是。

```
Failed setting prop "nodeType" on <div>: value 1 is invalid. TypeError: Cannot set property nodeType of #<Node> which has only a getter
```

**Comment by tazyong** [comment]
看了源码，这里处理得不好，所有属性都透传了。
但是在使用过程中，我们尽量避免使用原生 dom 的属性名，像 `HTMLElement`、`Element`、`Node` 里的属性都尽可能不要使用，如果必须使用可以加载前缀 `_`、`$` 来规避属性冲突。
[nodeType](https://developer.mozilla.org/zh-CN/docs/Web/API/Node/nodeType) 是 `Node` 类里的属性。
